.github/scripts/overwrite_changeset_changelog.py

```diff
@@ -22,39 +26,37 @@
 CHANGELOG_PATH = os.environ.get("CHANGELOG_PATH", "CHANGELOG.md")
 VERSION = os.environ['VERSION']
 PREV_VERSION = os.environ.get("PREV_VERSION", "")
-NEW_CONTENT = os.environ['NEW_CONTENT']
-
-def overwrite_changelog_section(content: str):    
-    """Replace a specific version section in the changelog content.
-    
-    Args:
-        content: The full changelog content as a string
-        
-    Returns:
-        The updated changelog content with the new section
-        
-    Example:
-        >>> content = "## 1.2.0\\nOld changes\\n## 1.1.0\\nOld changes"
-        >>> NEW_CONTENT = "New changes"
-        >>> overwrite_changelog_section(content)
-        '## 1.2.0\\nNew changes\\n## 1.1.0\\nOld changes'
-    """
+NEW_CONTENT = os.environ.get("NEW_CONTENT", "")
+
+def overwrite_changelog_section(changelog_text: str, new_content: str):
```

Comment:
Anyway we can write a test for this? And maybe possible to convert this into a TS file?

Mainly wondering if we have any other python dependencies - if not, i think it is worth to make this in TS and write a test for it as well.

Reply:
won't need python dependencies, this runs in Github Actions, I find it much easier than a TS file